components/global/global_header.scss

```
@@ -160,27 +160,43 @@
             cursor: pointer;
 
             .status {
-                background-image: linear-gradient(0deg, var(--sidebar-text-08), var(--sidebar-text-08));
+                background-image:
+                    linear-gradient(
+                        0deg,
+                        rgba(var(--global-header-text-rgb), 0.08),
+                        rgba(var(--global-header-text-rgb), 0.08)
+                    );
             }
         }
 
         &.active {
             background: var(--sidebar-text-16);
```

@nevyangelova, apologies, but I noticed one more variable that should probably be chaned while we are "under the hood".

```suggestion
            background:rgba(var(--global-header-text-rgb), 0.16);
```